releases/release-1.20/release-notes-draft.md

```diff
@@ -7,13 +7,14 @@
 
 ### (No, really, you MUST read this before you upgrade)
 
-- The deprecated flag --experimental-kustomize is now removed from kubeadm commands. Use --experimental-patches instead, which was introduced in 1.19. Migration infromation available in --help description for --exprimental-patches. ([#94871](https://github.com/kubernetes/kubernetes/pull/94871), [@neolit123](https://github.com/neolit123))
+- Windows hyper-v container featuregate is deprecated in 1.20 and will be removed in 1.21 ([#95505](https://github.com/kubernetes/kubernetes/pull/95505), [@wawa0210](https://github.com/wawa0210)) [SIG Node and Windows]
  
 ## Changes by Kind
 
 ### Deprecation
 
 - Action-required: kubeadm: graduate the "kubeadm alpha certs" command to a parent command "kubeadm certs". The command "kubeadm alpha certs" is deprecated and will be removed in a future release. Please migrate. ([#94938](https://github.com/kubernetes/kubernetes/pull/94938), [@yagonobre](https://github.com/yagonobre)) [SIG Cluster Lifecycle]
+- Action-required: kubeadm: remove the deprecated feature --experimental-kustomize from kubeadm commands. The feature was replaced with --experimental-patches in 1.19. To migrate see the --help description for the --experimental-patches flag. ([#94871](https://github.com/kubernetes/kubernetes/pull/94871), [@neolit123](https://github.com/neolit123)) [SIG Cluster Lifecycle]
```

Comment:
hmm somehow `94871` got moved from "Urgent Upgrade Notes" to "Deprecation".

I'm not sure if I understand the behavior of `action_required: true` in the PR map, I'm also not sure what qualifies as "Urgent Upgrade Notes" vs "Deprecation", since `94871` sounds like potential regression which users should know if they're still using the deprecated flag.

cc: @puerco

Reply:
If it's removing something that's already deprecated, is it still a deprecation? 🤔 Did it get miscategorised because it's got the word "deprecated" in the description?

Reply:
https://github.com/kubernetes/kubernetes/labels?q=action
`Denotes a PR that introduces potentially breaking changes that require user action.`

ideally 94871 should be under:
> (No, really, you MUST read this before you upgrade)

Reply:
should we edit/move this manually for now?